cli/internal/ui/ui.go

```diff
@@ -50,7 +44,7 @@ func rgb(i int) (int, int, int) {
 
 // Rainbow function returns a formated colorized string ready to print it to the shell/terminal
 func Rainbow(text string) string {
-	if !IsTTY {
+	if color.NoColor {
```

Comment:
I think I would prefer if the `applyForcedColor` function returned the value of `color.NoColor` as a `bool` instead of leaking this implementation detail as an implementation detail. This approach would only make sense if we stick with the current implementation where the value is only checked once on construction of `ColoredUi`.

Reply:
While we don't technically need to check this, the code is doing a non-trivial amount of formatting looping over each character passed in so I can go either way, but I like not having to check color.NoColor and by removing it we can prevent that abstraction leak as well.